src/Catalog/NuGet.Services.Metadata.Catalog.csproj

```diff
@@ -269,7 +272,7 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="NuGet.Packaging.Core">
-      <Version>4.8.0</Version>
+      <Version>5.0.0-preview1.5665</Version>
```

Comment:
What feature do you need from this unreleased/preview version of `NuGet.Packaging.Core`? Is it good to include preview package versions?

Reply:
That specific version is to align with [Gallery](https://github.com/NuGet/NuGetGallery/blob/0ca0368afbfc1bbf7133e2d72389fc168d35f6ce/src/NuGetGallery/NuGetGallery.csproj#L2362-L2367). It is needed to have access to the `PathUtility.StripLeadingDirectorySeparators` method.

In Gallery that particular version is used because it is one of the latest preview versions that has license metadata parsing support and not yet targeted to .NET Framework 4.7.2.